client/js/shout.js

```diff
@@ -205,6 +207,7 @@ $(function() {
 			"action",
 		].indexOf(type) !== -1) {
 			switch (type) {
+			case "invite": data.msg.formattedAction = "invited you to"; break;
```

Comment:
So, is this just for invites that someone sends to you. What I was referring to was invites that are sent to someone else. Will this be both of them? If so, the wording doesn't work for all of them.